go/vt/vtgate/planbuilder/testdata/filter_cases.json

```diff
@@ -923,10 +925,10 @@
   },
   {
     "comment": "Single table equality route with val arg",
-    "query": "select id from user where name = :a",
+    "query": "select id from user where name = 'a'",
```

Comment:
any reason for changing this query? This changes the test case

Reply:
thanks , restored the test case with https://github.com/vitessio/vitess/pull/17342/commits/b99ebabe960b49792dd426509d68fbad73a3fda3